build.sbt

```diff
@@ -6476,7 +6476,7 @@ createStdLibsIndexes := {
   DistributionPackage.indexStdLibs(
     stdLibVersion = targetStdlibVersion,
     ensoVersion   = ensoVersion,
-    stdLibRoot    = distributionRoot / "lib",
+    stdLibRoot    = distributionRoot / "lib" / "Standard",
```

Comment:
- well, root is still `lib`
- `Standard` is a namespace folder shared by all our libraries
- however I am advocating for us to also start producing non-standard libraries